cmd/removeEnumerator.go

```diff
@@ -109,3 +117,74 @@ func (s *directoryStack) Pop() (*azfile.DirectoryURL, bool) {
 		return &e, true
 	}
 }
+
+// TODO move after ADLS/Blob interop goes public
+// TODO this simple remove command is only here to support the scenario temporarily
+// Ultimately, this code can be merged into the newRemoveBlobEnumerator
+func removeBfsResource(cca *cookedCopyCmdArgs) (successMessage string, err error) {
+	ctx := context.Background()
+
```

Comment:
We don't support wildcards, right? e.g. * at the end of the URL (or inside the filename at the end of the URL). Is there an error message somewhere, if one is provided, to say that we don't support them?

Reply:
Added a check for better error message, thanks